package.json

```diff
@@ -171,44 +171,41 @@
     "ulid": "2.3.0",
     "unixify": "1.0.0",
     "update-notifier": "7.3.1",
-    "uuid": "9.0.1",
+    "uuid": "11.0.5",
     "wait-port": "1.1.0",
     "write-file-atomic": "5.0.1",
     "ws": "8.18.0",
-    "zod": "3.23.8"
+    "zod": "3.24.1"
   },
   "devDependencies": {
     "@babel/preset-react": "7.26.3",
     "@netlify/eslint-config-node": "7.0.1",
     "@netlify/functions": "3.0.0",
     "@sindresorhus/slugify": "2.2.1",
-    "@types/fs-extra": "11.0.4",
     "@types/inquirer": "9.0.7",
-    "@types/jsonwebtoken": "9.0.7",
+    "@types/jsonwebtoken": "9.0.8",
     "@types/lodash": "4.17.15",
     "@types/node": "22.10.7",
     "@types/node-fetch": "2.6.12",
     "@types/parse-gitignore": "1.0.2",
     "@types/prettyjson": "0.0.33",
     "@types/semver": "7.5.8",
-    "@types/uuid": "9.0.8",
     "@types/ws": "8.5.14",
     "@vitest/coverage-v8": "1.6.0",
-    "c8": "9.1.0",
+    "c8": "10.1.3",
     "cheerio": "1.0.0",
     "eslint-plugin-sort-destructure-keys": "2.0.0",
     "eslint-plugin-workspace": "file:./tools/lint-rules",
     "form-data": "4.0.1",
-    "fs-extra": "11.2.0",
-    "husky": "8.0.3",
-    "is-ci": "3.0.1",
-    "nock": "13.5.6",
+    "husky": "9.1.7",
+    "is-ci": "4.1.0",
+    "nock": "14.0.0",
```

Comment:
requires node 18 (like we do): https://github.com/nock/nock/releases/tag/v14.0.0